DynamoLeagueBlazor/Tests/Enums/PositionTests.cs

```diff
@@ -0,0 +1,30 @@
+﻿using DynamoLeagueBlazor.Shared.Enums;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DynamoLeagueBlazor.Tests.Enums
+{
+    public class PositionTests
+    {
+        [Theory]
+        [InlineData("QB", 1,0,1)]
+        [InlineData("QB", 1, 5, 150)]
+        [InlineData("RB", 1, 0, 1)]
+        [InlineData("RB", 1,5,300)]
+        [InlineData("WR", 1, 0, 1)]
+        [InlineData("WR", 1,5,150)]
+        [InlineData("TE", 1, 0, 1)]
+        [InlineData("TE", 1,5,145)]
+        [InlineData("DEF", 1, 0, 1)]
+        [InlineData("DEF", 1,5,6)]
```

Comment:
Just FYI this seems like overkill. I think there’s really only two pieces of behavior worth targeting here - that it should grab from the contract table when the chartValue is less, and vice versa. This is fine just muddies up the behavior under test.